frontend/kesaseteli/employer/src/pages/application.tsx

```diff
@@ -38,7 +39,11 @@ const ApplicationPage: NextPage = () => {
 
   if (!applicationId) {
     void router.replace(`${locale}/`);
-    return null;
+    return <PageLoadingSpinner />;
+  }
+  if (application?.status === 'submitted') {
```

Comment:
enum?

Reply:
It only has type but I think it's good enough. Enums are cumbersome imho:
```
type Status =
  | 'draft'
  | 'submitted'
  | 'additional_information_requested'
  | 'additional_information_provided'
  | 'accepted'
  | 'rejected'
  | 'deleted_by_customer';
```